Formula/mapnik.rb

```diff
@@ -1,9 +1,8 @@
 class Mapnik < Formula
   desc "Toolkit for developing mapping applications"
   homepage "http://www.mapnik.org/"
-  url "https://github.com/mapnik/mapnik/archive/v3.0.9.tar.gz"
-  sha256 "f0242606096e2c4ca2cd0caac1ff0fd5f8054a38b5f288ba38b0e397b5b311b2"
-  revision 1
+  url "https://github.com/mapnik/mapnik/releases/download/v3.0.12/mapnik-v3.0.12.tar.bz2"
```

Comment:
@zmwangx actually this is correct as-is. We like to use the release tarballs if they're available. Usually the problem is the reverse. We have to switch to the archive tarball temporarily for some reason, and then when that reason goes away, people forget to restore the release tarball until one of the maintainers notices there is a release tarball available and that it works again, and puts it back.